pages/about.tsx

```diff
@@ -122,15 +124,77 @@ export default function AboutPage() {
                 <MuiMarkdown>{content.joinUs.body}</MuiMarkdown>
               </Typography>
               <Stack sx={{ direction: 'column' }}>
-                <Button href="/donate" variant="contained" sx={{ width: '15.5rem' }}>{content.buttons[3].name}</Button>
-                <Button href="/get-involved" variant="contained" sx={{ width: '15.5rem' }}>{content.buttons[4].name}</Button>
+                <Button href="/donate" variant="contained" sx={{ width: '15.5rem' }}>{content.buttons[5].name}</Button>
+                <Button href="/get-involved" variant="contained" sx={{ width: '15.5rem' }}>{content.buttons[6].name}</Button>
               </Stack>
             </Grid>
           </Grid>
         </Box>
       </SectionContainer>
 
-      {aboutContent.ourTeam.map((item) => (
+      <SectionContainer id="our-partners" sx={{ margin: 'auto', mb: 4, maxWidth: '65rem' }} maxWidth={false}>
+        <Typography variant="h2" sx={{ my: '3rem' }}>{content.ourPartners.header}</Typography>
+        <PaperSection title="" sx={{ pt: 0, pb: 1 }}>
+          <Grid container>
+            <Grid item xs={12}>
+              <Box display="flex" justifyContent="center" sx={{ pt: 4 }}>
+                <ImageContainer
+                  alt=""
+                  src={content.ourPartners.imgsrc as string}
+                  width={700}
+                  height={450}
+                  useImageDimensions
+                  style={{
+                    height: '124px',
+                    width: '221px',
+                    objectFit: 'contain',
+                  }}
+                />
+              </Box>
+            </Grid>
+            <Grid
+              item
+              xs={12}
+              sx={{
+                display: 'flex', flexDirection: 'column', justifyContent: 'center', pt: '0.5rem',
+              }}
+            >
+              <Typography
+                sx={{
+                  color: '#2F3554',
+                  textAlign: 'center',
+                  fontSize: '20px',
```

Comment:
I think that most of these styles should not be hardcoded in, but rather, pull from the theme, which is already imported and existing in this file. You are already specifying the variant. When I delete the font size, font weight, line height, and letter spacing, I am left with almost the same result. Let's clean this up a little bit.